### ISSUE TITLE
Trigger function for marking block-height-duplicative headers

### ISSUE THREAD

**Issue by i-norden** [vulcanize/ipld-eth-db#58]
Write a trigger function for `eth.header_cids`, anytime a header is inserted with a blockheight that already exists in the table it either alters a (new) column in `eth.header_cids` or inserts a record to a new table for tracking duplicative heights.

**Comment by abdulrabbani00** [comment]
@i-norden - Is there a preference for either adding a new column or a new table? I feel like a new table can be more insightful and have more information (if we actually care for it). A few interesting columns to see in a new table might be:

1. block_number
2. []node_id - Not unique, we might want to know how many times a single node tried processing the same block.
3. []timestamp - Keep track of everytime "time" we tried to process the blocks (might "seem" more useful than it actually is).
4. count - The number of times we tried to track this block (although this might be redundant if we have a list of non-unique node_id).

If we wanted a simple, minimal solution, we can add a columns to `eth.header_cids` table, maybe something like `eth.header_cids(duplicative_block_height)`. And if we have 3 separate entries for a given block_heigh, each entry would have `duplicative_block_height:3`.

@i-norden - LMK what you think, initially I liked the idea of a separate table but it might not be as useful as I'm making it out to be.
